journal/modernized/02-chapter-1.adoc

```diff
@@ -82,14 +82,16 @@ A young man, a merchant, then present, who frequented my company,
 and would often speak of the people called Quakers,
 and say their principles were the truth,
 hearing the priest boast of his abusive behaviour to the said two women,
-answered and said.
-It was a poor victory he had gotten over two poor women;
+answered and said,
+"`It was a poor victory he had gotten over two poor women;`"
+//There aren't quotations in the PDF, but this seems like a direct quote to me. Also on line 94
```

Comment:
Hmmm.... tough call. It kinda depends if his friend was talking to William or to the priest.  Because if he was talking to the priest (which is likely) it would be "It is a poor victory you have gotten over two poor women."  I would kinda lean towards leaving the quotation marks as you put them, but changing it to how I have it written here.